packages/ckeditor5-table/src/tableutils.js

```diff
@@ -94,15 +94,22 @@ export default class TableUtils extends Plugin {
 	 * @param {Number} [options.columns=2] The number of columns to create.
 	 * @param {Number} [options.headingRows=0] The number of heading rows.
 	 * @param {Number} [options.headingColumns=0] The number of heading columns.
+	 * @param {String} [options.defaultProperties={}] The default properties for the created table.
```

Comment:
```suggestion
	 * @param {Object} [options.defaultProperties={}] The default properties for the created table.
```

Reply:
Shouldn't be it named just `attributes` or `tableAttributes` (to distinguish it from cell attributes)? Looking at the code this option is specific to the default properties feature.

Reply:
+1 for `tableAttributes`. The word "default" means nothing in Table Utils' context.

Reply:
I'd go with the `tableProperties` name to align the option to the plugin. Same for cell properties.

Reply:
But it's hard to be consistent :/

Reply:
Fixed in c7b23f2.